backend/src/modules/boards/services/update.board.service.ts

```diff
@@ -273,7 +273,7 @@ export default class UpdateBoardServiceImpl implements UpdateBoardService {
 							return {
 								text: cardItem.text,
 								votes: cardItem.votes,
-								createdByTeam: subBoard.title,
+								createdByTeam: subBoard.title.replace('board', ''),
```

Comment:
The word `board` don't have any space after?
If yes I think you need to change to:
```suggestion
								createdByTeam: subBoard.title.replace('board ', ''),
```